specification/transport_layer/transport_layer.tex

```diff
@@ -534,6 +545,15 @@ \subsection{State variables}
     or the protocol stack can estimate this value automatically by analyzing the transfer intervals. \\
 \end{UAVCANSimpleTable}
 
+The reason nodes are required to let the first interface time out before using the next one
```

Comment:
One more English nit:

Nodes are required to let the first interface timeout before using the next one because the transfer ID field is expected to wrap around frequently...